knowledge_graph/routes.py

```diff
@@ -188,10 +188,33 @@ def get_personal_values():
 
 @app.route("/get_actions", methods=["POST"])
 def get_actions():
+    """Temporary test function to take a JSON full of user scores and calculate the
+    best nodes to return to a user. Will be deprecated and replaced by /feed.
+
+    """
     try:
         scores = request.json
     except:
         return make_response("Invalid JSON"), 400
     recommended_nodes = get_user_nodes(scores)
     response = Response(dumps(recommended_nodes))
     return response, 200
+
+
+@app.route("/feed", methods=["POST"])
+def get_feed():
+    """The front-end needs to request personalized climate change effects that are most
+    relevant to a user to display in the user's feed.
+
+    """
+    session_id = str(request.args.get("session-id"))
+    try:
+        scores = db.session.query(Scores).filter_by(session_id=session_id).first()
+    except:
+        return make_response("Invalid Session ID or No Information for ID")
+
+    scores = scores.__dict__
+    del scores["_sa_instance_state"]
+    recommended_nodes = get_user_nodes(scores)
+    response = Response(dumps(recommended_nodes))
```

Comment:
Is this dumping everything about the node stored in the networkx object? For now, the front end only want the node titles for the top recommended nodes. There's more information on the node stored in networkx than the frontend will need for the app. The extra info is there just in case it's needed later. So the front end from my understanding just wants the node title for now.